lib/pltr/v2/reducers/beats.js

```diff
@@ -31,9 +33,9 @@ import { clone } from 'lodash'
 //  - bookId: Number,
 //  - "series": String literal,
 
-const add = addNode('id')
+const add = tree.addNode('id')
 
-const defaultBeats = add(newTree('id'), null, defaultBeat)
+const defaultBeats = add(tree.newTree('id'), null, defaultBeat)
```

Comment:
We call `tree.newTree('id')` enough that I think it might be worth binding `const newTree = tree.newTree(`id`)` along side the `addNode` binding.